test/unit-test/ConfigFiles/portmacro.h

```diff
@@ -1,6 +1,6 @@
 /*
- * FreeRTOS Kernel V10.3.0
- * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * FreeRTOS Kernel V10.4.3
+ * Copyright 2020 Cambridge Consultants Ltd.
```

Comment:
mmm i thought it should be the reverse Amazon.com instead of Cambridge Consultants ..

Reply:
Removed the file